roles/logstash/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for docker
+- name: restart zookeeper
```

Comment:
seems like this should use the global 'handlers' role then and the code should move there, as it confused me being here